libs/algorithms.py

```diff
@@ -6,13 +6,35 @@
 #
 
 
-def bubbleSort(list):
-    return list
+def bubbleSort(ls):
+    is_sorted = False
+    while not is_sorted:
+        for i in range(len(ls)):
+            next_element = None
+            try:
+                next_element = ls[i+1]
+            except IndexError:
+                break
+            if ls[i] > next_element:
+                temp_i = int(ls[i+1])
+                ls[i+1] = int(ls[i])
```

Comment:
`int()` feels redundant in these lines but it is crucial. `int` is a type that needs to be constructed using the element. I did this to avoid referencing it. We wouldn't have this problem if python was as cool as cpp and let us use pointers.